x-pack/solutions/observability/plugins/slo/server/services/get_slo_health.ts

```diff
@@ -193,17 +172,20 @@ function getTransformHealth(
 
 function computeHealth(
   transformStatsById: Dictionary<TransformGetTransformStatsTransformStats>,
-  item: { sloId: string; sloInstanceId: string; sloRevision: number }
+  item: { id: string; instanceId: string; revision: number }
 ): { overall: 'healthy' | 'unhealthy'; rollup: HealthStatus; summary: HealthStatus } {
-  const rollup = getTransformHealth(
-    transformStatsById[getSLOTransformId(item.sloId, item.sloRevision)]
-  );
+  const rollup = getTransformHealth(transformStatsById[getSLOTransformId(item.id, item.revision)]);
   const summary = getTransformHealth(
-    transformStatsById[getSLOSummaryTransformId(item.sloId, item.sloRevision)]
+    transformStatsById[getSLOSummaryTransformId(item.id, item.revision)]
   );
 
   const overall: 'healthy' | 'unhealthy' =
-    rollup.status === 'healthy' && summary.status === 'healthy' ? 'healthy' : 'unhealthy';
+    rollup.status === 'healthy' &&
```

Comment:
💬 Includes transformState for computing the overall status.

Reply:
We have a related [issue](https://github.com/elastic/kibana/issues/238018) for including transformState to the overall health status. 

Not related to the current issue, just wanted to bring into your attention the discussions around if the SLO "needs attention", when the SLO is `disabled`, and either or both of its transforms are `not stopped`. Let's continue these discussions on [this](https://github.com/elastic/kibana/issues/238018) issue.